newIDE/app/src/ResourcesList/LocalJfxrBridge.js

```diff
@@ -35,22 +36,19 @@ export const openJfxr = ({
 
   ipcRenderer.removeAllListeners('jfxr-changes-saved');
   ipcRenderer.on('jfxr-changes-saved', (event, newFilePath, fileMetadata) => {
-
     const resourceName = path.relative(projectPath, newFilePath); // TODO: move into a generic createOrUpdateResource function that piskel can also use in app/src/ResourcesList/ResourceUtils.js
-    const resourcesManager = project.getResourcesManager();
-    if (resourcesManager.hasResource(resourceName)) {
-      resourcesManager.removeResource(resourceName)
-    }
-    const audioResource = new gd.AudioResource();
-    audioResource.setFile(resourceName);
-    audioResource.setName(resourceName);
-    resourcesManager.addResource(audioResource);
-    audioResource.delete();
-
-    const newMetadata = {
-      jfxr: fileMetadata,
-    };
-    resourcesManager.getResource(resourceName).setMetadata(JSON.stringify(newMetadata));
+    createOrUpdateResource(project, new gd.AudioResource(), resourceName);
+
+    const newMetadata = fileMetadata.data
```

Comment:
What is the purpose of this extra condition?

Reply:
Its an extra check if there is any data in there, so as to avoid creating an object containing pskl:{} , but I will check again if this is needed

Reply:
@4ian when the user saves in piskel and there is only one layer, metadata is not passed back to gdevelop. This is the reason I am checking it, but for jfxr the user always saves the metadata - so you are right - I dont need it there xD It was silly to copy the behavior over to jfxr